laravel/resources/views/layout.blade.php

```diff
@@ -1,19 +1,22 @@
 <!doctype html>
 <html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>@yield('site_title') | {{ config('app.name', 'TeamSpeak Dynamic Banner') }}</title>
-    @include('layouts.head')
-</head>
-<body class="d-flex flex-column vh-100">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
 
-@yield('dataTables_config')
+        <title>@yield('site_title') | {{ config('app.name', 'TeamSpeak Dynamic Banner') }}</title>
```

Comment:
[IMPROVEMENT]

Instead of defining the default `TeamSpeak Dynamic Banner` here, I would rather update it in the respective configuration file: https://github.com/Sebbo94BY/teamspeak-dynamic-banner/blob/main/laravel/config/app.php#L18